src/org/thoughtcrime/securesms/WebViewActivity.java

```diff
@@ -156,8 +157,8 @@ public void onResume() {
 
   @Override
   protected void onDestroy() {
-    super.onDestroy();
     webView.destroy();
+    super.onDestroy();
```

Comment:
this is unrelated just because it seemed to be more correct, in theory I guess you should first destroy the webview before calling this activity's destroy because then maybe the rest of the code maybe is not executed? but I am not sure how this works